presto-main/src/main/resources/webapp/src/components/QueryList.jsx

```diff
@@ -63,13 +63,25 @@ export class QueryListItem extends React.Component {
         }
 
         let formattedQueryText = "";
+        let currentLineCount = 0;
```

Comment:
`lineCount` might look slightly better? Wdyt?

Reply:
yeah as long as it says "**count**" in it, it implies that its current so we could make it shorter